File: examples/FaxLineCreateExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Dropbox.Sign.Api;
using Dropbox.Sign.Client;
using Dropbox.Sign.Model;

namespace Dropbox.SignSandbox;

public class FaxLineCreateExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.Username = "YOUR_API_KEY";

        var faxLineCreateRequest = new FaxLineCreateRequest(
            areaCode: 209,
            country: FaxLineCreateRequest.CountryEnum.US
        );

        try
        {
            var response = new FaxLineApi(config).FaxLineCreate(
                faxLineCreateRequest: faxLineCreateRequest
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling FaxLineApi#FaxLineCreate: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: examples/FaxLineCreateExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.FaxLineApi();
apiCaller.username = "YOUR_API_KEY";

const faxLineCreateRequest: models.FaxLineCreateRequest = {
  areaCode: 209,
  country: models.FaxLineCreateRequest.CountryEnum.Us,
};

apiCaller.faxLineCreate(
  faxLineCreateRequest,
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling FaxLineApi#faxLineCreate:");
  console.log(error.body);
});
